db/migrate/20161229080315_add_comments_count_to_events.rb

```diff
@@ -0,0 +1,18 @@
+class AddCommentsCountToEvents < ActiveRecord::Migration
+  class TempComment < ActiveRecord::Base
+    self.table_name = 'comments'
+  end
+
+  class TempEvent < ActiveRecord::Base
+    self.table_name = 'events'
+  end
+
+  def change
+    add_column :events, :comments_count, :integer, default: 0, null: false
+
+    TempEvent.all.each do |event|
```

Comment:
`TempEvent.all.each` will load ALL events into memory, which is potentially bad.
It's better to use [`TempEvent.find_each`](http://api.rubyonrails.org/v4.2.5/classes/ActiveRecord/Batches.html#method-i-find_each), which will load the events in batches.